library_generation/owlbot/.gitignore

```diff
@@ -0,0 +1 @@
+google-java-format.jar
```

Comment:
@blakeli0 FYI we ended up reverting the use of `fmt:format` because the format plugin is inherited from `google-cloud-jar-parent`, which is fixed _after_ postprocessing is run.

Reply:
Can you please elaborate? Why can't we use `mvn fmt:format` in the end after the poms are generated?

Reply:
Using `fmt` gives a missing versions [error](https://paste.googleplex.com/6024683240030208)

That's because these versions come from a parent pom not yet fixed (the fix occurs [here](https://github.com/googleapis/sdk-platform-java/blob/d2367889d753ddc25185aa2e47cb40be1ebb2a4f/library_generation/new_client/new-client.py#L379-L386), after postprocessing).

Reply:
One way to fix this is the change the template; or move the format to the last step.

Reply:
This jar file is removed.